cli/cmd/initialize/init.go

```diff
@@ -98,7 +130,7 @@ func InitCmd(cfg *config.Config) *cobra.Command {
 	initCmd.Flags().BoolVar(&listExamples, "list-examples", false, "List available example projects")
 	initCmd.Flags().StringVar(&exampleName, "example", "default", "Name of example project")
 	initCmd.Flags().Lookup("example").NoOptDefVal = "default" // Allows "--example" without a specific name
-	initCmd.Flags().StringVar(&projectPath, "project", ".", "Project directory")
+	initCmd.Flags().BoolVar(&interactive, "interactive", true, "Non Interactive mode")
```

Comment:
Change to `"Prompt for missing required parameters"`

Reply:
Let's make this a root-level flag and set it on `cfg` instead? It will be handy in many commands. You don't have to implement support for it everywhere, just add it there for now.